resolver_conn_wrapper_test.go

```diff
@@ -70,6 +69,10 @@ func TestParseTargetString(t *testing.T) {
 		{"google.com", resolver.Target{"", "", "google.com"}},
 		{"google.com/?a=b", resolver.Target{"", "", "google.com/?a=b"}},
 		{"/unix/socket/address", resolver.Target{"", "", "/unix/socket/address"}},
+
+		// If we can only parse part of the target.
+		{"://", resolver.Target{"", "", "://"}},
```

Comment:
Is it worth testing other partial matches like `"[a]:[b]"` and `"[a]:/[b]"`?

Reply:
Done